presto-main/src/test/java/com/facebook/presto/util/TestTimeZoneUtils.java

```diff
@@ -87,6 +87,12 @@ public void testNamedZones()
             DateTimeZone indexedZone = getDateTimeZone(TimeZoneKey.getTimeZoneKey(zoneId));
 
             assertDateTimeZoneEquals(zoneId, indexedZone);
+            if (dateTimeZone.getID().equals("Pacific/Kanton")) {
```

Comment:
Can we add context why this is being excluded?

Reply:
can this be moved above with the other zoneId checks?

Reply:
No, it can't. There's an extra weirdness with this particular timezone. If you start with Pacific/Enderbury as the zoneId. getDateTimeZone will return the new name Pacific/Kanton so this check needs to come after line 87.